control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -1397,6 +1402,29 @@ func (r *reconciler) reconcileUserCertCABundle(ctx context.Context, hcp *hyperv1
 	return nil
 }
 
```

Comment:
let's follow up to document the func with the same PR rationale. So is easier to reason about future relocation of the logic across components